tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -658,18 +666,45 @@ <h2>Continue...</h2>
           const state = checked ? 'break' : 'disable';
           // TODO(cais): Investigate why this is fired twice sometimes.
           this._requestBreakpointStateChange(
-              tf_debugger_dashboard.getCleanNodeName(debugObject.node_name),
+              tf_debugger_dashboard.getCleanNodeName(
+                  debugObject.device_name + '/' + debugObject.node_name),
               debugObject.output_slot, debugObject.debug_op, state);
         });
       },
+      _focusOnGraphNode(deviceName, nodeName) {
+        if (deviceName != null &&
+            this._activeRuntimeGraphDeviceName !== deviceName) {
+          this.set('_activeRuntimeGraphDeviceName', deviceName);
+        }
+        this._setTopRightRuntimeGraphsToActive();
+        this.$$('#graph').set('selectedNode', nodeName);
+      },
       _createListNodeClickedHandler() {
         return((deviceName, nodeName) => {
-          if (deviceName != null &&
-              this._activeRuntimeGraphDeviceName !== deviceName) {
-            this.set('_activeRuntimeGraphDeviceName', deviceName);
+          this._focusOnGraphNode(deviceName, nodeName);
+        });
+      },
+      _createFeedFetchTargetClickedHandler() {
+        return((graphElementName) => {
+          let nodeName = graphElementName;
+          // If it is a tensor name, get the node name.
+          if (nodeName.indexOf(':') !== -1) {
```

Comment:
Optional: Maybe compute once and reuse value.

Reply:
This computation is pretty cheap. So I'd rather save the space.